code/modules/modular_computers/computers/item/computer.dm

```diff
@@ -576,7 +577,7 @@
 			enabled = TRUE
 
 /// Sets visible messages to also send to holder because coders didn't know it didn't do this
-/obj/item/modular_computer/visible_message(message, self_message, blind_message, vision_distance = DEFAULT_MESSAGE_RANGE, list/ignored_mobs, visible_message_flags = NONE)
+/obj/item/modular_computer/visible_message(message, self_message, blind_message, vision_distance, list/ignored_mobs, visible_message_flags)
```

Comment:
?

Reply:
this is done in the parent proc so it's unnecessary here, and is what i meant by "code improvements"

Reply:
Doesn’t seem like an improvement tbh, just makes it more obtuse to read